week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -12,12 +12,18 @@ let bottle = {
   volume: 0,
   fill: function() {
     // calling this function should make you bottles volume = 100;
+    this.volume = 100;
   },
   drink: function() {
     // calling this function should decrease your bottles volume by 10;
+    this.volume -= 10;
   },
   empty: function() {
     // this function should return true if your bottles volume = 0

```

Comment:
This is technically doing what the question asked, but I think implicit in this is that it should return `false` if volume _isn't_ `0` - can you do that too? And can you try to do so by making the function _smaller_ rather than _bigger_? (i.e. without adding an `else` clause?)